haec-user-service/src/modules/user/services/user.service.ts

```diff
@@ -46,14 +46,51 @@ export class UserService {
           email: input.email,
         },
       });
-
       if (await bcrypt.compare(input.password, user.hashedPass)) {
         return true;
       }
     } catch (error) {
       console.error(error);
     }
-
     return false;
   }
+
+// This will log the user ** IN **
+
+  public async loginUser(input: {
+    email: string;
+    password: string;
+  }): Promise<string> {
+    const verified = await this.verifyUser({ email: input.email, password: input.password})
+    if (verified == true){
+      // Generates random 21 character token with Crypto
+      const Crypto = require('crypto')
```

Comment:
Fyi, `require` statements are an "old style" of bringing in libraries -- the "new" way is using `import`. I _think_ the way to do this would be `import * from crypto as Crypto` but you may need to look up an example. 

Other than changing this to an import, should also be at the top ☝️ of the file.

Reply:
Gotcha, revamped it and ready to go.